### DISCUSSION TITLE
[repro provided] adding text paragraph without parsing the children into type="text"

### DISCUSSION THREAD

**Discussion by addlistener** [GrapesJS/grapesjs#5814] [Q&A] [answered]
https://stackblitz.com/edit/grapesjs-react-custom-ui-mtdfh3?file=src%2FApp.tsx

I'm trying to add the following to the canvas
```
        <div data-gjs-type="text">
          <p>
            <a href="https://vimeo.com/317860394">Neighborhood Golf Association</a> from <a href="https://vimeo.com/user2412496">HELLER films</a> on <a href="https://vimeo.com">Vimeo</a>.
          </p>
          <p>For the past 10 years, street photographer Patrick Barr aka Tiger Hood has become a local legend known for bringing golf to the streets of NYC. It&rsquo;s a game that requires only three items: a golf club, a newspaper-stuffed milk carton, and a crate. What was initially just a way for Barr to pass time has gained traction from major news outlets and celebrities on a global scale. However, street golf seems to overshadow his true passion&hellip; photography. Barr&rsquo;s archive consists of thousands of mind blowing film photographs of NYC from the 1990&rsquo;s to 2000&rsquo;s. His goal was to preserve a time and place that he predicted would dissolve in the coming years. With his archive as evidence, he predicted correctly.<br />
          <br />
          &ldquo;Neighborhood Golf Association&rdquo; is a short documentary by long time friend, Nicolas Heller aka New York Nico. The 12-minute film explores a day in the life of Barr supplemented by archive footage collected over their three-year friendship. <br />
          <br />
          </p>
        </div>
```

However child nodes are parsed into type="text" or type="link". Is it possible we only preserve the top level type="text" node? Since it's more reasonable we edit the paragraph as a single text box(plus I have a custom rte)
<img width="905" alt="image" src="https://github.com/GrapesJS/grapesjs/assets/4531670/e54f86c1-180d-4a15-84da-ecd374c88229">

**Comment by artf** [comment, marked as answer]
Did you try this option (`config.domComponents.disableTextInnerChilds`)?
https://github.com/GrapesJS/grapesjs/blob/717e4297ce3010866af0b14976755bae3e929540/src/dom_components/config/config.ts#L17-L28

**Reply by addlistener** [reply]
Thank you! It works!

And furthermore, is there any place I can trigger my `customRte.getContent`? My customRte adds some additional classes to links 

<img width="524" alt="image" src="https://github.com/GrapesJS/grapesjs/assets/4531670/4ec3a8f6-dbce-41e5-bd9f-cc393cc0105a">

<img width="547" alt="image" src="https://github.com/GrapesJS/grapesjs/assets/4531670/d8b062c0-2c2c-42d4-9a05-c0d073bc8c67">

**Reply by artf** [reply]
Not sure what you want to trigger... that sounds like the responsibility of your custom RTE, not the editor itself. Try to explore its configuration to disable the behavior you don't need.
